Lib/multiprocessing/managers.py

```diff
@@ -1159,16 +1159,21 @@ def __imul__(self, value):
         self._callmethod('__imul__', (value,))
         return self
 
+    __class_getitem__ = classmethod(types.GenericAlias)
+
 
-DictProxy = MakeProxyType('DictProxy', (
+BaseDictProxy = MakeProxyType('DictProxy', (
     '__contains__', '__delitem__', '__getitem__', '__iter__', '__len__',
     '__setitem__', 'clear', 'copy', 'get', 'items',
-    'keys', 'pop', 'popitem', 'setdefault', 'update', 'values'
+    'keys', 'pop', 'popitem', 'setdefault', 'update', 'values',
     ))
-DictProxy._method_to_typeid_ = {
-    '__iter__': 'Iterator',
+class DictProxy(BaseDictProxy):
+    _method_to_typeid_ = {
+        '__iter__': 'Iterator',
     }
 
+    __class_getitem__ = classmethod(types.GenericAlias)
```

Comment:
I'd prefer it if we followed the style used elsewhere in this module, which has the happy side-effect of meaning that this PR has a smaller diff :)

https://github.com/python/cpython/blob/d0dcd27d3a8b8680803510fee59836cd732feb7a/Lib/multiprocessing/managers.py#L1178-L1193

```suggestion
BaseDictProxy._method_to_typeid_ = {
    '__iter__': 'Iterator',
}


class DictProxy(BaseDictProxy):
    __class_getitem__ = classmethod(types.GenericAlias)
```